apps/core/README.md

```diff
@@ -30,7 +30,14 @@ module.exports = {
 };
 ```
 
-### Debugging in VS Code
+## Add plugins
+
+You can add plugins to the core by adding them to the plugins folder. Each plugin live in its own folder and have at least a `index.ts` file.
+
+The plugin folder is located by default in `apps/core/src/plugins` or `apps/core/dist/plugins` when compiled.
+It can be configured with the `PLUGINS_PATH` environment variable or the `pluginsPath` variable in the `config/local.js` file. ⚠️ The path must be under the `apps/core/src/plugins` folder (eg. `/plugins/my-own-repo/my-plugins`)
```

Comment:
**suggestion:** 
```suggestion
It can be configured with the `PLUGINS_PATH` environment variable or the `pluginsPath` variable in the `config/local.js` file. ⚠️ The path must be under the `apps/core/src/plugins` folder (eg. `apps/core/src/plugins/my-own-repo/my-plugins`)
```